CHANGELOG.rst

```diff
@@ -5,6 +5,24 @@ community.zabbix Release Notes
 .. contents:: Topics
 
 
+v1.9.1
+======
+
+Major Changes
```

Comment:
minor, major is reserved for backwards incompatible i think

Reply:
do I change it via antsibul--changelog or directly  in CHANGELOG.rst?

Reply:
I alywas updated either changelog/fragments/XYZ or changelog/changelog.yml and re-run antsibull-changelog